### ISSUE TITLE
Do we need to talk about source address selection for CLAT?

### ISSUE THREAD

**Issue by philsbln** [furry13/v6ops-464xlat-enable#38]
I only stumbled over this issue as macOS, if there a multiple prefixes in the same RA as the PREF64 option, prefers ULA over GUA as CLAT IP address, which broke some things at home for me.

If we follow the source address selection in rfc6724 or draft-ietf-6man-rfc6724-update, the well-know prefix or any non-ULA custom prefix would match ::/0 , the same-label rule would require the source for the packages leaving the v6 side of the CLAT should be a GUA.
On the other hand, the CLAT address is not chosen per flow, but on creation of the CLAT, so what does not apply directly. 

A way around this issue is following the model in draft-link-v6ops-gulla, so by putting the PREF64 only on the GUA or ULA RA would allow the operator to choose.

I suspect we should talk about how we expect this to behave…

**Comment by oskar456** [comment]
I agree that if PREF64 does not lie in the ULA space, ULA address should definitely not be preferred for setting up CLAT. I have experienced the same issue and even reported it to Apple. I think using RFC 6724 is a good and sensible approach.

**Comment by furry13** [comment]
closed by #45
